I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00gamemenu.rpy", line 170, in script
    $ ui.interact()
  File "renpy/common/00gamemenu.rpy", line 170, in <module>
    $ ui.interact()
  File "game/imaginary_friends_screens.rpy", line 145, in execute
    screen statsUI():
  File "game/imaginary_friends_screens.rpy", line 145, in execute
    screen statsUI():
  File "game/imaginary_friends_screens.rpy", line 149, in execute
    frame:
  File "game/imaginary_friends_screens.rpy", line 155, in execute
    vbox:
  File "game/imaginary_friends_screens.rpy", line 187, in execute
    frame:
  File "game/imaginary_friends_screens.rpy", line 195, in execute
    viewport id "vp_stats":
  File "game/imaginary_friends_screens.rpy", line 197, in execute
    vbox:
  File "game/imaginary_friends_screens.rpy", line 205, in execute
    for key, v in friends_stats:
  File "game/imaginary_friends_screens.rpy", line 205, in execute
    for key, v in friends_stats:
  File "game/imaginary_friends_screens.rpy", line 205, in <module>
    for key, v in friends_stats:
ValueError: too many values to unpack (expected 2)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00gamemenu.rpy", line 170, in script
    $ ui.interact()
  File "C:\Users\Beller\Desktop\Saved Files\renpy-8.0.3-sdk\renpy\ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Beller\Desktop\Saved Files\renpy-8.0.3-sdk\renpy\python.py", line 1061, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00gamemenu.rpy", line 170, in <module>
    $ ui.interact()
  File "C:\Users\Beller\Desktop\Saved Files\renpy-8.0.3-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\Beller\Desktop\Saved Files\renpy-8.0.3-sdk\renpy\display\core.py", line 3377, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\Beller\Desktop\Saved Files\renpy-8.0.3-sdk\renpy\display\core.py", line 3810, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "C:\Users\Beller\Desktop\Saved Files\renpy-8.0.3-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Beller\Desktop\Saved Files\renpy-8.0.3-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Beller\Desktop\Saved Files\renpy-8.0.3-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Beller\Desktop\Saved Files\renpy-8.0.3-sdk\renpy\display\screen.py", line 451, in visit_all
    callback(self)
  File "C:\Users\Beller\Desktop\Saved Files\renpy-8.0.3-sdk\renpy\display\core.py", line 3810, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "C:\Users\Beller\Desktop\Saved Files\renpy-8.0.3-sdk\renpy\display\screen.py", line 462, in per_interact
    self.update()
  File "C:\Users\Beller\Desktop\Saved Files\renpy-8.0.3-sdk\renpy\display\screen.py", line 653, in update
    self.screen.function(**self.scope)
  File "game/imaginary_friends_screens.rpy", line 145, in execute
    screen statsUI():
  File "game/imaginary_friends_screens.rpy", line 145, in execute
    screen statsUI():
  File "game/imaginary_friends_screens.rpy", line 149, in execute
    frame:
  File "game/imaginary_friends_screens.rpy", line 155, in execute
    vbox:
  File "game/imaginary_friends_screens.rpy", line 187, in execute
    frame:
  File "game/imaginary_friends_screens.rpy", line 195, in execute
    viewport id "vp_stats":
  File "game/imaginary_friends_screens.rpy", line 197, in execute
    vbox:
  File "game/imaginary_friends_screens.rpy", line 205, in execute
    for key, v in friends_stats:
  File "game/imaginary_friends_screens.rpy", line 205, in execute
    for key, v in friends_stats:
  File "game/imaginary_friends_screens.rpy", line 205, in <module>
    for key, v in friends_stats:
ValueError: too many values to unpack (expected 2)

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.3.22090809
Imaginary Friends 1.0
Sun Apr 23 11:49:49 2023
